lib/liquid/combinators/tags/decrement.ex

```diff
@@ -23,6 +23,8 @@ defmodule Liquid.Combinators.Tags.Decrement do
   import NimbleParsec
   alias Liquid.Combinators.Tag
 
+  @type t :: [decrement: [variable_name: String.t()]]
```

Comment:
Shouldn't a Decrement type be just a tagged tuple instead of a list of tagged tuples?

Reply:
Every tag is a keyword list (which is a list of tagged tuples)